### ISSUE TITLE
VisualState is not working

### ISSUE THREAD

**Issue by nomurayoshuak** [adospace/reactorui-maui#229]
@adospace 
Hi, VisualState is not working in the my project and even sample app.
https://github.com/adospace/reactorui-maui/tree/main/samples/MauiReactor.TestApp

- The background color of the label is not Aqua.
- The background color does not change to Aqua when I click the button.

It may not be a MauiReactor issue, but is there a solution?
I run app on android emulator.

```c#
class CounterPage : Component<CounterPageState>
{
    public override VisualNode Render()
        => ContentPage("Counter Sample",
            VStack(
                Label($"Counter: {State.Counter}")
                    .AutomationId("Counter_Label")
                    .VisualState("CommonStates", "Normal", MauiControls.Label.BackgroundColorProperty, Colors.Aqua),

                Button("Click To Increment", () => SetState(s => s.Counter++))
                    .AutomationId("Counter_Button")
                    .VisualState("CommonStates", "Pressed", MauiControls.Button.BackgroundColorProperty, Colors.Aqua)
            )
            .Spacing(10)
            .Center()
        );
    
}
```

**Comment by adospace** [comment]
have you tried with a .NET MAUI classic project? I usually use the common state Selected for the collection view and it appears to be working

**Comment by nomurayoshuak** [comment]
What is a classic project? Does it mean a single project?

If there is a problem with VisualState, I would like to achieve the following. Is there an alternative in MauiReactor?

- Change the style of the background color etc. when tapping the button (visual feedback for tapping)
- Similarly, visual feedback for taps other than buttons (Frame, Label, etc.)

It likes this.

https://github.com/adospace/reactorui-maui/assets/57802609/12fc725e-80ce-40f5-aae6-0f5b5090c9ec

**Comment by adospace** [comment]
Fixed in version 2.0.36, updated documentation as well:
https://adospace.gitbook.io/mauireactor/components/controls/button
